CHANGES.md

```diff
@@ -1,6 +1,38 @@
 
 # Changes
 
+## edge-21.1.2
+
+This edge release continues the work on decoupling non-core Linkerd components.
+Commands that use the viz-extension i.e, `dashboard`, `edges`, `routes`,
+`stat`, `tap` and `top` are moved to the `viz` sub-command. These commands are still
+available under root but are marked deprecated and will be removed in a
+later stable release.
+
+This release also features proxy's dependencies upgrade to the
+Tokio v1 ecosystem.
+
+* Moved sub-commands that use the viz-extension under `viz`
+* Started ignoring pods with status.phase=Succeeded when watching IP addresses
+  in destination. This is useful for re-use of IPs of terminated pods
+* Support Bring your own Jaeger use-case by adding `collector.jaegerAddr` in
+  the jaeger extension.
+* Fixed an issue with the generation of working manifests in the
+  `podAntiAffinity` use-case
+* Added support for the modification of proxy resources in the viz
+  extension through `values.yaml` in Helm and flags in CLI.
+* Improved error reporting for port-forward logic with namespace
+  and pod data, used across dashboard, checks, etc
+  (thanks @piyushsingariya)
+* Added support to disable the rendering of `linkerd-viz` namespace
+  resource in the viz extension (thanks @nlamirault)
+* Made service-profile generation work offline with `--ignore-cluster`
+  flag (thanks @piyushsingariya)
+* Proxy's Tap API is disabled by default and it is enabled only when
+  `LINKERD2_PROXY_TAP_SVC_NAME` configuration is set. This means that
+  `LINKERD2_PROXY_TAP_DISABLED` is no longer honored
```

Comment:
I'm back and forth on if this should be included? If users are adding these values and then manually injecting, then maybe this is _user facing_, but I doubt that's very common.

Also, it may be a little confusing because users can actually still disable tap, but that's through an annotation (`config.linkerd.io/disable-tap`) on the pod or the namespace.

Since nothing about the annotation has changed, I think maybe it's okay to leave this out, but definitely not a blocker!

Reply:
+1 I don't think this is user-facing.

Reply:
Removed this